pkg/network/route_cache.go

```diff
@@ -275,13 +275,20 @@ func (n *netlinkRouter) Route(source, dest util.Address, netns uint32) (Route, b
 		})
 
 	if err != nil {
-		routeCacheTelemetry.netlinkErrors.Inc()
+		var tag string
 		if iifIndex > 0 {
-			if errno, ok := err.(syscall.Errno); ok && (errno == syscall.EINVAL || errno == syscall.ENODEV) {
-				// invalidate interface cache entry as this may have been the cause of the netlink error
-				n.removeInterface(source, netns)
+			errno, ok := err.(syscall.Errno)
+			if ok {
+				tag = errno.Error()
+				if errno == syscall.EINVAL || errno == syscall.ENODEV {
+					// invalidate interface cache entry as this may have been the cause of the netlink error
+					n.removeInterface(source, netns)
+				}
 			}
 		}
+		log.Warnf("Netlink Error: %s", err)
```

Comment:
We should put more context in the message; just "netlink error" is too generic.

Reply:
lmk if the new message is enough